lib/src/shops/models/address_model.dart

```diff
@@ -5,7 +5,7 @@ part 'address_model.g.dart';
 @JsonSerializable()
 class ShopAddress {
   Map<String, dynamic> geometry;
-  String city;
+  String? city;
   String countryCode;
```

Comment:
Others can't be null as well?

Reply:
I guess so but not sure, will make them nullable